net/wifi-presence/files/wifi-presence.init

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+START=95
+# stops before networking stops
+STOP=89
+
+PROG=/usr/bin/wifi-presence
+CONF=wifi-presence
+
+start_service() {
+        # Load config.
+        config_load "${CONF}"
+
+        local apName
+        local debounce
+        local hassAutodiscovery
+        local hassPrefix
+        local hostapdSocks
+        local mqttAddr
+        local mqttID
+        local mqttUsername
+        local mqttPassword
+        local mqttPrefix
+        local sockDir
+        local verbose
+
+        config_get apName main apName
+        config_get debounce main debounce
+        config_get hassAutodiscovery main hassAutodiscovery
+        config_get hassPrefix main hassPrefix
+        config_get hostapdSocks main hostapdSocks
+        config_get mqttAddr main mqttAddr
+        config_get mqttID main mqttId
+        config_get mqttUsername main mqttUsername
+        config_get mqttPassword main mqttPassword
+        config_get mqttPrefix main mqttPrefix
+        config_get sockDir main sockDir
+        config_get_bool verbose main verbose
+
+        procd_open_instance
+
+        procd_set_param command ${PROG}
+        [[ ! -z "${apName}" ]] && procd_append_param command "-apName=${apName}"
```

Comment:
umm no. Run through shellcheck.

All of these should be using single brackets and -n instead of -z